### PR TITLE
Update to Preview 4 of mvvm toolkit

### PR DESCRIPTION
**Pull request by jamesmontemagno**

### CODE REVIEW

File: Part 2 - MVVM/README.md [hunk 1]
```diff
@@ -421,10 +421,10 @@ We will use an `ObservableCollection<Monkey>` that will be cleared and then load
     }
     ```
 
-    However, with the .NET Community Toolkit we simply can add the `[ICommand]` attribute to our method:
+    However, with the .NET Community Toolkit we simply can add the `RelayCommand` attribute to our method:
 
     ```csharp
-     [ICommand]
+     RelayCommand
```
Comment (michael-hawker) [inline Part 2 - MVVM/README.md:427]: Brackets lost here.

File: Part 3 - Navigation/MonkeyFinder/ViewModel/MonkeysViewModel.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@ public MonkeysViewModel(MonkeyService monkeyService)
         this.monkeyService = monkeyService;
     }
 
-    [ICommand]
+    RelayCommand
```
Comment (michael-hawker) [inline Part 3 - Navigation/MonkeyFinder/ViewModel/MonkeysViewModel.cs:15]: Brackets lost here too.

**Review by michael-hawker — CHANGES_REQUESTED** [review]
For some reason many of the `[ICommand]` lost their brackets when changing to `RelayCommand` instead of `[RelayCommand]`. (More than just the two I called out here, realized it was more prevalent.)

File: Part 3 - Navigation/MonkeyFinder/ViewModel/MonkeysViewModel.cs (hunk 2 above)
Comment (jamesmontemagno) [reply Part 3 - Navigation/MonkeyFinder/ViewModel/MonkeysViewModel.cs:15]: Whoops!

File: Part 2 - MVVM/README.md [hunk 3]
```diff
@@ -438,12 +438,12 @@ We will use an `ObservableCollection<Monkey>` that will be cleared and then load
     {
         /// <summary>The backing field for <see cref="GetMonkeysCommand"/>.</summary>
         [global::System.CodeDom.Compiler.GeneratedCode("CommunityToolkit.Mvvm.SourceGenerators.ICommandGenerator", "8.0.0.0")]
-        private global::CommunityToolkit.Mvvm.Input.AsyncRelayCommand? getMonkeysCommand;
-        /// <summary>Gets an <see cref="global::CommunityToolkit.Mvvm.Input.IAsyncRelayCommand"/> instance wrapping <see cref="GetMonkeysAsync"/>.</summary>
+        private global::CommunityToolkit.Mvvm.Input.Async[RelayCommand]? getMonkeysCommand;
```
Comment (jamesmontemagno) [inline Part 2 - MVVM/README.md:441]:
```suggestion
        private global::CommunityToolkit.Mvvm.Input.AsyncRelayCommand? getMonkeysCommand;
```

File: Part 2 - MVVM/README.md [hunk 4]
```diff
@@ -438,12 +438,12 @@ We will use an `ObservableCollection<Monkey>` that will be cleared and then load
...
+        /// <summary>Gets an <see cref="global::CommunityToolkit.Mvvm.Input.IAsync[RelayCommand]"/> instance wrapping <see cref="GetMonkeysAsync"/>.</summary>
```
Comment (jamesmontemagno) [inline Part 2 - MVVM/README.md:442]:
```suggestion
        /// <summary>Gets an <see cref="global::CommunityToolkit.Mvvm.Input.IAsyncRelayCommand"/> instance wrapping <see cref="GetMonkeysAsync"/>.</summary>
```

File: Part 2 - MVVM/README.md [hunk 5]
```diff
@@ -438,12 +438,12 @@ We will use an `ObservableCollection<Monkey>` that will be cleared and then load
...
         [global::System.CodeDom.Compiler.GeneratedCode("CommunityToolkit.Mvvm.SourceGenerators.ICommandGenerator", "8.0.0.0")]
         [global::System.Diagnostics.DebuggerNonUserCode]
         [global::System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
-        public global::CommunityToolkit.Mvvm.Input.IAsyncRelayCommand GetMonkeysCommand => getMonkeysCommand ??= new global::CommunityToolkit.Mvvm.Input.AsyncRelayCommand(new global::System.Func<global::System.Threading.Tasks.Task>(GetMonkeysAsync));
+        public global::CommunityToolkit.Mvvm.Input.IAsync[RelayCommand] GetMonkeysCommand => getMonkeysCommand ??= new global::CommunityToolkit.Mvvm.Input.Async[RelayCommand](new global::System.Func<global::System.Threading.Tasks.Task>(GetMonkeysAsync));
```
Comment (jamesmontemagno) [inline Part 2 - MVVM/README.md:446]:
```suggestion
        public global::CommunityToolkit.Mvvm.Input.IAsyncRelayCommand GetMonkeysCommand => getMonkeysCommand ??= new global::CommunityToolkit.Mvvm.Input.AsyncRelayCommand(new global::System.Func<global::System.Threading.Tasks.Task>(GetMonkeysAsync));
```

**Review by michael-hawker — APPROVED** [review]

**Pull request merged**